spec/controllers/odk_form_download_spec.rb

```diff
@@ -18,54 +18,58 @@
       @form3.publish!
     end
 
-    describe 'listing forms' do
-      it 'should succeed' do
-        get_s(odk_form_list_path)
+    describe "listing forms" do
+      it "should succeed" do
+        get_s(odk_form_list_path, format: :xml)
```

Comment:
It's not adding a `.xml`, it's setting content type and/or accepts headers -- if we hardcoded the paths we'd still need the `format: :xml` there